### PR TITLE
User can update the help request task

### PR DESCRIPTION
**Pull request by Carltesio**
# [PT Story](https://www.pivotaltracker.com/story/show/172292180)
```
As a person who is requesting for help
in order to create a help task that matches my needs
I would like to be able to update,change and see the total amount of my request
```
## Changes proposed in this pull request:
* Adds serializer for task model
* Adds PUT action to update order
## What I have learned working on this feature:
* Using active model serializer 
## Packages/Gems added
- gem 'active_model_serializers', '~> 0.10.2'
## Screenshots (Only if client side story)

### CODE REVIEW

File: app/models/task.rb [hunk 1]
```diff
@@ -1,3 +1,7 @@
 class Task < ApplicationRecord
     has_many :task_items
+
+    def order_total
+        task_items.joins(:product).sum("products.price")
```
Comment (blake-futchi) [inline app/models/task.rb]: edit spacing

File: app/serializers/task_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class TaskSerializer < ActiveModel::Serializer
+  attributes :id, :products, :total, :order_total
+
+  def products
+    object.task_items.group_by(&:product_id).map do |_key, value|
+      product = value.uniq(&:product_id)[0].product
+      { amount: value.size, name: product.name, total: (value.size * product.price) }
+    end
+	end
```
Comment (blake-futchi) [inline app/serializers/task_serializer.rb]: edit spacing

Comment (robin-lillqvist) [inline app/serializers/task_serializer.rb]: indentation

**Review by robin-lillqvist — CHANGES_REQUESTED** [review]

**Comment by robin-lillqvist** [comment]
Could you also included a screenshot of passing tests? so we know they are ;)

**Comment by pierre-1** [comment]
Looks good, based on the latest updates.

**Pull request merged**